### DISCUSSION TITLE
How to get markdown autocomplete with MDX?

### DISCUSSION THREAD

**Discussion by Zamiell** [mdx-js/mdx#2424] [Q&A] [answered]
When typing in a file called "foo.md", VSCode will automatically autocomplete this:

```
[foo](f
```

To this:

```
[foo](foo.md)
```

Similarly, VSCode will automatically autocomplete this:

```
[foo](foo.md#s
```

To this:

```
[foo](foo.md#some-arbitrary-header)
```

Both of these things do not happen in an MDX file by default. After installing this extension, I naively expended this functionality to be added, but it does not seem to be the case.
Is this a bug in the extension? Do I have to enable some additional setting in order to get this functionality?

**Comment by remcohaszing** [comment, marked as answer]
I am seeing path completions in MDX for links as you describe, but I need to press <kbd>Ctrl</kbd> + <kbd>Space</kbd> to trigger the auto complete. For markdown however, the completion pops up automatically based on certain trigger characters. Is that the behaviour you’re seeing too?

**Reply by Zamiell** [reply]
Yes, that is correct. I was not aware that `Ctrl` + `Space` would work in this way and find it highly surprising that it is necessary.
If possible, I propose that the behavior is unified such that explicitly typing `Ctrl` + `Space` in MDX files is unnecessary.

**Reply by remcohaszing** [reply]
I agree. I'll try to fix this soon. Feel free to [create an issue](https://github.com/mdx-js/mdx-analyzer/issues) to remind me.
